src/actionbar/index.css

```diff
@@ -0,0 +1,53 @@
+@import '../commons/index.css';
+
+:root {
+  --spectrum-actionbar-height: var(--spectrum-global-dimension-size-600);
+  --spectrum-actionbar-padding-x: var(--spectrum-global-dimension-size-200);
+  --spectrum-actionbar-margin-x: var(--spectrum-global-dimension-size-200);
+  --spectrum-actionbar-offset-y: var(--spectrum-global-dimension-size-200);
+
+  --spectrum-actionbar-min-width: 280px;
+  --spectrum-actionbar-max-width: 1280px;
+}
+
+/* topdoc
+{{ actionbar/actionbar.yml }}
+*/
+.spectrum-ActionBar {
+  align-items: center;
+  flex-direction: initial;
+  justify-content: space-between;
+
+  height: var(--spectrum-actionbar-height);
+  padding: 0 var(--spectrum-actionbar-padding-x);
+  min-width: var(--spectrum-actionbar-min-width);
+  max-width: var(--spectrum-actionbar-max-width);
+  width: 100%;
+}
+
+/* topdoc
+{{ actionbar/table-actionbar.yml }}
+*/
+.spectrum-ActionBar-container {
+  margin: 0 var(--spectrum-actionbar-margin-x);
+
+  bottom: 0;
+  z-index: 1;
+
+  display: flex;
+  justify-content: center;
+
+  .spectrum-ActionBar {
+    &.is-open {
+       transform: translateY(calc(-1*(var(--spectrum-actionbar-height) + var(--spectrum-actionbar-offset-y))));
+    }
+  }
+
+  &.spectrum-ActionBar-sticky {
+    position: sticky;
+  }
+
+  &.spectrum-ActionBar-fixed {
```

Comment:
Assuming we go with the suggestion I made above, since these are [modifiers](https://github.com/suitcss/suit/blob/master/doc/naming-conventions.md#ComponentName--modifierName) (not children), they should be double dashes. And, since they're not states (no `.is-` prefix), they don't need to be nested and defined with `&.`:

Move them to the top level and rename them as:

```
.spectrum-ActionBar--sticky {  ... }
.spectrum-ActionBar--fixed {  ... }
```